src/OpenCensus.Exporter.Stdout/StdoutTraceExporter.cs

```diff
@@ -0,0 +1,82 @@
+﻿// <copyright file="StdoutTraceExporter.cs" company="OpenCensus Authors">
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenCensus.Exporter.Stdout
+{
+    using OpenCensus.Exporter.Stdout.Implementation;
+    using OpenCensus.Trace.Export;
+
+    /// <summary>
+    /// Exporter of Open Census traces to Zipkin.
+    /// </summary>
+    public class StdoutTraceExporter
```

Comment:
Let's rename it to StdoutExporter. In future we may want to extend it to export logs and metrics